doc/syntax.rst

```diff
@@ -46,6 +46,17 @@ includes macros, functions, struct and union members, enumerations, etc.
 Documentation comments followed by comments (documentation or not) are included
 as normal paragraphs in the order they appear.
 
+Empty Documentation Comments
+----------------------------
+
+Use the empty documentation comment ``/***/`` to only generate the C or C++
+Domain directives without documentation. This can be useful for documenting
+struct members or enumeration constants without actually documenting the
+enclosing struct or enum.
+
+It's also a way to let Sphinx know about types to highlight them in
+e.g. parameter documentation.
+
```

Comment:
I wonder whether this is necessary at all, but I don't object to it.

Reply:
I can leave that part out. I was contemplating leaving out the entire documentation on empty comments, but thought there's no harm.

Reply:
I meant the whole section too by the way. I see no point in documenting this at all, I think anyone would expect this behaviour intuitively.

Reply:
Maybe just one sentence in the previous section?

"Empty documentation comments `/***/` create the C or C++ Domain directives without documentation."

Or nothing?

Reply:
Up to you... I don't feel it's needed but I'd prefer a single sentence to a whole section.

Reply:
Dropped it completely. Can be added later if needed.